source/flow.rst

```diff
@@ -599,6 +598,67 @@ Fields:
 * **Name** *(optional)*: Name of the node
 
 
+Publish in FTP topic
+********************
+
+.. _kafka2ftp_node:
+.. image:: images/nodes/kafka2ftp_node.png
+    :width: 20%
+    :align: left
+    :alt: kafka2ftp_node
+
+Node to forwarding messages to Apache Kafka FTP topic.
+
+It publish to the topic ``tenant.dojot.ftp`` (*tenant* is defined by which tenant the flow belongs to)
+where in which messages are produced with information about the file name,
```

Comment:
where in which messages are produced with information about the file name, -> in which the messages are produced with informations about the file name,